src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -111,7 +111,7 @@ Status CoreWorkerMemoryStore::Put(const ObjectID &object_id, const RayObject &ob
   std::unique_lock<std::mutex> lock(lock_);
   auto iter = objects_.find(object_id);
   if (iter != objects_.end()) {
-    return Status::KeyError("object already exists");
+    return Status::ObjectExists("object already exists in the memory store");
```

Comment:
If we want to introduce a new `ObjectExists` status, we should propagate this status all the way up to `ObjectStoreInterface:Put`. And also fix the behavior of `PlasmaStoreProvider` as well. 
But for now, I think it's fine to just return `Status::OK()` here. Because we don't need to handle this status, other than ignoring it.

Reply:
Yes it's probably ideal to just propagate up to `ObjectStoreInterface`, which can ignore the object exists error. But I think the current way of ignoring it in `MemoryStoreProvider` is still better than doing it in `MemoryStore`, because for a layered design, the store shouldn't be aware of how the upper layer handles a specific error.